migration/inferencing-migration/migrate-service.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+set -e
+
+subscription_id="<SUBSCRIPTION_ID>"
+resource_group="<RESOURCEGROUP_NAME>"
+workspace_name="<WORKSPACE_NAME>"
+v1_service_name="<SERVICE_NAME>" # name of your aci/aks service
+local_dir="<LOCAL_PATH>"
+online_endpoint_name="<NEW_ENDPOINT_NAME>"
+online_deployment_name="<NEW_DEPLOYMENT_NAME>"
+
+migrate_type="Managed"
+
+# STEP1 Export services
+echo 'Exporting services...'
+output=$(python3 export-service-util.py --export --export-json -w $workspace_name -g $resource_group -s $subscription_id -sn $v1_service_name| tee /dev/tty)
+read -r storage_account blob_folder v1_compute < <(echo "$output" |tail -n1| jq -r '"\(.storage_account) \(.blob_folder) \(.v1_compute)"')
+
+# STEP2 Download template & parameters files
+echo 'Downloading files...'
+az storage blob directory download -c azureml --account-name "$storage_account" -s "$blob_folder" -d $local_dir --recursive --subscription $subscription_id --only-show-errors 1> /dev/null
+
+# STEP3 Overwrite parameters
+echo 'Overwriting parameters...'
+echo
+params_file="$local_dir/$blob_folder/$v1_compute/$migrate_type/$v1_service_name.params.json"
+template_file="$local_dir/$blob_folder/online.endpoint.template.json"
+output=$(python3 export-service-util.py --overwrite-parameters -mp "$params_file" -me "$online_endpoint_name" -md "$online_deployment_name"| tee /dev/tty)
+params=$(echo "$output"|tail -n1)
+
+# STEP4 Deploy to MIR
```

Comment:
I propose we place MIR with something like managed online endpoint. MIR is an internal name.